tests/spread/general/classic-patchelf-override/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: Build a classic snap and validates elf patching
+
+systems:
+  - -ubuntu-20.04
+  - -ubuntu-20.04-64
+  - -ubuntu-20.04-amd64
+  - -ubuntu-20.04-arm64
+  - -ubuntu-20.04-armhf
+  - -ubuntu-20.04-s390x
+  - -ubuntu-20.04-ppc64el
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "./snap/snapcraft.yaml"
+
+  apt-get install patchelf dpkg-dev -y
+  apt-mark auto patchelf dpkg-dev
+
+restore: |
+  cd snap
+  snapcraft clean
+  rm -f ./*.snap
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snapcraft.yaml"
+
+execute: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  base="$(get_base)"
+
+  cd snap
+  cat snapcraft.yaml
+  snapcraft prime
+
+  arch_triplet="$(dpkg-architecture -q DEB_HOST_MULTIARCH)"
+
+  RPATH_ORIGIN_MATCH="^\\\$ORIGIN/../usr/lib/$arch_triplet:/snap/$base/current/lib/$arch_triplet"
+
+  # Verify typical binary.
+  patchelf --print-interpreter prime/bin/hello | MATCH "^/snap/$base/current/lib.*ld.*.so.*"
```

Comment:
Shouldn't this check that patchelf did not run (by matching against `"^/lib.*/ld.*.so.*"`)?